apps/accounts/api/v1/serializers.py

```
@@ -0,0 +1,106 @@
+            "username",
+            "email",
+            "password",
+            "img_url",
+            "about",
+            "pscore",
+        )
+        extra_kwargs = {
+            "password": {"required": False},
+        }
+
+    def update(self, instance, validated_data):
```

Ruff (0.8.2)

Missing return type annotation for public function `update`

(ANN201)